src/proxy-ui-api/src/main/java/org/niis/xroad/securityserver/restapi/converter/MessageLogEncryptionStatusConverter.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * The MIT License
+ * Copyright (c) 2019- Nordic Institute for Interoperability Solutions (NIIS)
+ * Copyright (c) 2018 Estonian Information System Authority (RIA),
+ * Nordic Institute for Interoperability Solutions (NIIS), Population Register Centre (VRK)
+ * Copyright (c) 2015-2017 Estonian Information System Authority (RIA), Population Register Centre (VRK)
+ * <p>
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * <p>
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ * <p>
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.niis.xroad.securityserver.restapi.converter;
+
+import ee.ria.xroad.common.MessageLogEncryptionStatusDiagnostics;
+
+import org.niis.xroad.securityserver.restapi.openapi.model.MessageLogEncryptionMember;
+import org.niis.xroad.securityserver.restapi.openapi.model.MessageLogEncryptionStatus;
+import org.springframework.stereotype.Component;
+
+import java.util.stream.Collectors;
+
+@Component
+public class MessageLogEncryptionStatusConverter {
+    public MessageLogEncryptionStatus convert(
+            MessageLogEncryptionStatusDiagnostics messageLogEncryptionStatusDiagnosticsStatus) {
```

Comment:
arg messageLogEncryptionStatusDiagnosticsStatus name seems to be off, perhaps messageLogEncryptionStatusDiagnostics ?